pandas/plotting/_core.py

```diff
@@ -3115,19 +3115,42 @@ def kde(self, bw_method=None, ind=None, **kwds):
 
     def area(self, x=None, y=None, **kwds):
         """
-        Area plot
+        Draw an stacked area plot.
+
+        An area plot displays quantitative data visually.
+        This function wraps the matplotlib area function.
 
         Parameters
         ----------
-        x, y : label or position, optional
-            Coordinates for each point.
-        `**kwds` : optional
+        x : label or position, optional, default None
```

Comment:
you can leave out the ", default None" part (saying it is "optional" should be informative enough"). 

But we should specify here what it used for `x` by default. I suppose this is the index?

(and the same comment for `y`

Reply:
Yes I believe it's the index by default.